app/src/main/res/values-night/themes.xml

```diff
@@ -1,6 +1,6 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.Multimatum" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+    <style name="Theme.Multimatum" parent="Theme.MaterialComponents.DayNight.NoActionBar">
```

Comment:
What is this about? Maybe notify this change in the PR.

Reply:
Good point! As discussed with members of the team last week, I removed the "Action Bar" (top purple bar) to get more space to look more like a recent app.